audeer/core/utils.py

```diff
@@ -573,6 +574,52 @@ def _worker():
     return results
 
 
+def sort_versions(
+        versions: List[str],
+) -> List:
+    """Sort version numbers.
+
+    If the version starts with ``'v'``
```

Comment:
This sounds like versions starting with `v` are not taken into account.

Reply:
Agree, would the following work:

```rst
If the version starts with ``'v'``,
the ``'v'`` is ignored during sorting.
```

Reply:
Yes! But maybe rephrase to `If a version...`

Reply:
Updated now to

![image](https://user-images.githubusercontent.com/173624/107354892-08012780-6acf-11eb-9c0f-e9cea9838527.png)

Reply:
Should I remove the `'` around the `v`?

Reply:
I'd say it's fine like this.

Reply:
Ok, actually in other places we don't put it in `'`

Reply:
So yes, maybe remove it then.

Reply:
Done